_posts/2021-12-02-PC-Engines-Firmware-v4-15-0-1.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.15.0.1"
+date:   2021-12-02
```

Comment:
```suggestion
date:   2021-12-03
```

Reply:
Done, also renamed the file and updated the release date in index.html.